frontend/src/libs/components/image-selector/libs/helpers/read-file.helper.ts

```diff
@@ -0,0 +1,12 @@
+const readFile = (
+  file: File,
+  onFinishCallback: (url: string) => void,
```

Comment:
```suggestion
  onFinish: (url: string) => void,
```
it has the prefix `on`, which means that it's a callback.